src/util.ts

```diff
@@ -10,9 +10,8 @@ import crc32 from 'buffer-crc32';
 import crc from 'crc';
 import { sendMail } from '@/mailer';
 import { config, disabledFeatures } from '@/config-manager';
-import { TokenOptions } from '@/types/common/token-options';
-import { Token } from '@/types/common/token';
-import { IPNID, IPNIDMethods } from '@/types/mongoose/pnid';
+import { getSystemTypeFromValue, getTokenTypeFromValue, OAuthTokenGenerationResponse, OAuthTokenOptions, SystemType, SystemTypes, Token, TokenOptions, TokenTypes } from '@/types/common/token';
+import { HydratedPNIDDocument, IPNID, IPNIDMethods } from '@/types/mongoose/pnid';

```

Comment:
These aren't using `import type`, I thought the eslint stuff added that? Or am I mistaken?

Reply:
Eslint is an open PR, when this all gets merged eslint will tidy it up

Reply:
Perfect, I thought that was already brought in so I was just confused in that regard :+1: I thought maybe eslint just wasn't working for some reason